src/libraries/Microsoft.PowerFx.Core/Public/Values/ColorValue.cs

```diff
@@ -23,7 +24,7 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
-            throw new System.NotImplementedException();
+            sb.Append($"RGBA({Value.R},{Value.G},{Value.B},{Math.Round(Value.A / 255.0, 3)})");
```

Comment:
> 3 [](http://example.com/codeflow?start=87&length=2)

Does this lose precision? That may be unavoidable.

Reply:
It would, but I think precision of 3 would be good enough to show for alpha!